scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -127,25 +126,32 @@ public static ObjectMapper createObjectMapper() {
     return mapper;
   }
 
+  /** Attempt to parse naively, and force parsing using UTF-8 if that doesn't work */
   public static Optional<Document> parseXml(File file) {
     try {
-      DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+      return Optional.of(DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(file));
+    } catch (ParserConfigurationException | IOException | SAXException e) {
+      log.error("Not really a UTF-16 parsing exception, forcing UTF-8 parsing...", e);
+      DocumentBuilder builder;
+      try {
+        builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+      } catch (ParserConfigurationException e2) {
+        throw new RuntimeException(e2);
```

Comment:
I'm guessing it's not `ParserConfigurationException` that's thrown when the specified encoding is wrong, so you can probably move it to the fail-fast catch block and remove this inner try/catch.

Either way, you should only be throwing a `RuntimeException` here if you expect this to never happen, there should be an exception message that explains this, and you shouldn't be catching that exception below.